pkg/sfu/buffer/buffer.go

```diff
@@ -159,6 +167,16 @@ func (b *Buffer) Bind(params webrtc.RTPParameters, codec webrtc.RTPCodecCapabili
 	default:
 		b.codecType = webrtc.RTPCodecType(0)
 	}
+	for _, ext := range params.HeaderExtensions {
```

Comment:
This block can potentially be moved into the if block below which is video specific. There is already an extensions things for audio below. Or, maybe move the audio one out here so that all extensions handling is in one place.

Reply:
Changed